EnglishFilter/sections/specific.txt

```diff
@@ -144,6 +144,9 @@ kiss-anime.ws##script[id^="BB_SLOT_"] ~ div[id][style^="overflow: hidden; width:
 ! ##.banner
 aplayer2.me,viralizeit.net,sex3.com,softonic.com,3pornstarmovies.com,flvto.biz,govtsmartjob.com,cutearn.ca,dfporn.net,cll.press,hellporno.net,youramateurporn.com,getlink.pw,embedtvseries.com,keygames.com,uclick.in,flv2mp3.by,doodhwali.com,smutindia.com,dvdcover.com,xxxadultphoto.com,gabicoins.com,qpdownload.com,windows10portal.com,2conv.com,javcloud.com,underhentai.net,cutwin.com,hqasiananal.com,bradsknutson.com,flv2mp3.org,subscene.com,yourlink.in,iplayer.pw,qpornx.com,sofifa.com,jawcloud.co,indiansgetfucked.com,medi-bayreuth.de,oxforddictionaries.com,tube18.sex##.banner
 !
+flagandcross.com###klicked-shopify-widget
+flagandcross.com##.contextual-ad-d
```

Comment:
Is not hiding any left-over for me.

Reply:
@BlazDT 
![image](https://user-images.githubusercontent.com/8361299/61078004-4c769e00-a428-11e9-8363-0b19a5a9086e.png)

Reply:
That is #adtoniq-msgr-bar. When user presses on OK, it is possible he gets more ads, therefore English filter imho.